app/assets/stylesheets/sufia/_form-progress.scss

```diff
@@ -1,3 +1,25 @@
+@media screen and (min-width: 400px) {
+  .affix {
+    position: static;
+    margin-top: 1em;
+  }
+}
+@media screen and (min-width: 768px) {
+  aside.form-progress {
+    margin-right: 0.5em;
+  }
+  .affix {
+    position: fixed;
+  }
+  .tab-content {
+    min-height: 55em;
```

Comment:
Since I'm using CSS positioning, it pulls the save/visibility box out of the normal document flow. The minimum height of 55em ensures that the widget without the additional metadata fields doesn't dive behind the footer.